pkg/helm/handlers/handlers.go

```diff
@@ -68,11 +68,28 @@ func (h *HelmHandlers) HandleHelmList(user *auth.User, w http.ResponseWriter, r
 	conf := actions.GetActionConfigurations(h.ApiServerHost, ns, user.Token, &h.Transport)
 	resp, err := actions.ListReleases(conf)
 	if err != nil {
-		serverutils.SendResponse(w, http.StatusBadGateway, serverutils.ApiError{fmt.Sprintf("Failed to list helm releases: %v", err)})
+		serverutils.SendResponse(w, http.StatusBadGateway, serverutils.ApiError{Err: fmt.Sprintf("Failed to list helm releases: %v", err)})
 	}
 
 	w.Header().Set("Content-Type", "application/json")
 
 	res, _ := json.Marshal(resp)
 	w.Write(res)
 }
+
+func (h *HelmHandlers) HandleGetRelease(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	queryParams := r.URL.Query()
+	ns := queryParams.Get("ns")
+	chartName := queryParams.Get("release_name")
+
+	conf := actions.GetActionConfigurations(h.ApiServerHost, ns, user.Token, &h.Transport)
+	release, err := actions.GetRelease(chartName, conf)
+	if err != nil {
+		serverutils.SendResponse(w, http.StatusBadGateway, serverutils.ApiError{Err: fmt.Sprintf("Failed to list helm releases: %v", err)})
```

Comment:
and we should return here, otherwise, you are going to append more to your response.

Reply:
fixed https://github.com/openshift/console/pull/4059/commits/2f5c774264084098676056abb8e594cd10274217